bin/add-coursier.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eu
+set -o nounset
+
+PLUGINS_FILE="/home/jvican/.sbt/1.0/plugins/plugins.sbt"
```

Comment:
How will drone pick that up? Shouldn't it be in `/root/.sbt/...`?

Reply:
JEEZ, forgot to update it.